base/src/main/java/edu/artic/base/utils/DateTimeHelper.kt

```diff
@@ -27,4 +29,8 @@ class DateTimeHelper {
                 .appendPattern("MMMM d   h:mm a")
                 .toFormatter()
     }
-}
\ No newline at end of file
+}
+
+fun ZonedDateTime.getLocalDateTime(): ZonedDateTime {
```

Comment:
This is not returning a `LocalDateTime` which means its confusing. You mean `convertToLocalTime`?

Reply:
We are storing datetime in `UTC format` but need to convert it to the user's timezone before we dispaly. This is handy extension function to add updated timezone info to ZonedDateTime. Since it was confusing, I renamed it to `toCurrentTimeZone`.